pages/dkp/kommander/1.4/clusters/attach-cluster/requirements-for-attaching/index.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: layout.pug
+navigationTitle: Requirements for Attaching an Existing Cluster to Kommander
+title: Requirements for Attaching an Existing Cluster to Kommander
+menuWeight: 3
+excerpt: Requirements for attaching an existing cluster to Kommander
+beta: true
+---
+
+## Basic requirements
+
+You can attach existing clusters to Kommander whether or not those clusters have networking restrictions.
+
+For attaching existing clusters without networking restrictions, the requirements depend on the Kommander 1.X version that you plan to use. The supported Kubernetes versions are between version 1.16.x and 1.20.x with a maximum version of 1.20.5. To attach an existing cluster, the Kommander management cluster must be able to reach the services and the api-server of the target cluster.
+
+For attaching clusters _with_ networking restrictions, the lowest Kubernetes version should be less than or equal to version 1.19.x. For these types of clusters, the Kommander cluster uses a tunnel to access the clusters you plan to attach. Any managed service you want to expose in the Kommander cluster requires the creation of a reverse proxy, or the use of HTTPS_PROXY environment variables that point to the Kommander tunnel proxy server.
+
+When working with clusters that have networking restrictions, you need to run some YAML manifests on the existing cluster, so that Kommander can collect a resulting `kubeconfig` file used to establish the tunnel.
```

Comment:
```suggestion
When working with clusters that have networking restrictions, you will need to apply some additional YAML manifests on the existing cluster, so that Kommander can collect a resulting `kubeconfig` file used to establish the tunnel.
```

Reply:
We could link to here: 
/dkp/kommander/1.4/clusters/tunnel-cli/
or specifically - this starts the instructions about getting that manifest.yaml file here: 
/dkp/kommander/1.4/clusters/tunnel-cli/#connecting-a-remote-cluster

Reply:
I am indeed linking to that page in this area, and will be letting Jon G know that he _may_ be missing a couple of steps.